bar-trender-front-end/src/views/LandingPage.js

```diff
@@ -13,29 +12,25 @@ import {
 } from "reactstrap";
 
 // core components
-import ExamplesNavbar from "../../components/Navbars/ExamplesNavbar.js";
-import LandingPageHeader from "../../components/Headers/LandingPageHeader.js";
-import DefaultFooter from "../../components/Footers/DefaultFooter.js";
-
-import image_left from '../../assets/img/expositions/hU-kQ3Epxeq2dhaBpUgYfYaPhHEOKXnHXSeUqLjTygYBV05OHhUSZEWilh_Da9zkI1d_cgz91KIPevD_BBhBWhaKevognkx6Bv7-QwkQdRG9oznKG6wOae4avH8ksi6bkJBLWl4.png';
-import image_left_2 from '../../assets/img/expositions/Yn0xRl4G5E1eabgf9nyC9j6DVQVHd5DBNcPehVZwakLHYP-toRbW22a8kFesYK_taX0ZY_WviWVcT3bQ40tlKhaKSuAQAu6graIF.png';
-
-import image_right from '../../assets/img/expositions/TR9IDnSgMV79XktfRCxesUmLacTZJI9fb3Cv3-aMamIGyWdL_OagKWYcJJAPqgm62bjW9I6yHlMsOhowVROsAUiNui0CGo-qmPU-.png';
-
-import employee_1 from '../../assets/img/bg-landing.png';
-import employee_2 from '../../assets/img/victor.jpg';
-import employee_3 from '../../assets/img/enrique.jpg';
-import employee_4 from '../../assets/img/alvaro.jpg';
-import employee_5 from '../../assets/img/jose.jpg';
-import employee_6 from '../../assets/img/carlos-pardo.jpg';
-import employee_7 from '../../assets/img/alejandro.jpg';
-import employee_8 from '../../assets/img/xema.jpg';
-import employee_9 from '../../assets/img/miguel.jpg';
-import employee_10 from '../../assets/img/miguel-angel.jpg';
-import employee_11 from '../../assets/img/fran.jpg';
-
-
+import MainNavbar from "../components/Navbars/MainNavbar.js";
+import LandingPageHeader from "../components/Headers/LandingPageHeader.js";
+import DefaultFooter from "../components/Footers/DefaultFooter.js";
 
+// images
+import image_left from '../assets/img/expositions/hU-kQ3Epxeq2dhaBpUgYfYaPhHEOKXnHXSeUqLjTygYBV05OHhUSZEWilh_Da9zkI1d_cgz91KIPevD_BBhBWhaKevognkx6Bv7-QwkQdRG9oznKG6wOae4avH8ksi6bkJBLWl4.png';
+import image_left_2 from '../assets/img/expositions/Yn0xRl4G5E1eabgf9nyC9j6DVQVHd5DBNcPehVZwakLHYP-toRbW22a8kFesYK_taX0ZY_WviWVcT3bQ40tlKhaKSuAQAu6graIF.png';
+import image_right from '../assets/img/expositions/TR9IDnSgMV79XktfRCxesUmLacTZJI9fb3Cv3-aMamIGyWdL_OagKWYcJJAPqgm62bjW9I6yHlMsOhowVROsAUiNui0CGo-qmPU-.png';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7123646)
```suggestion
import image_right from "../assets/img/expositions/TR9IDnSgMV79XktfRCxesUmLacTZJI9fb3Cv3-aMamIGyWdL_OagKWYcJJAPqgm62bjW9I6yHlMsOhowVROsAUiNui0CGo-qmPU-.png";
```